### DISCUSSION TITLE
La vista compañía no se ve como la que sale en la documentacion

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#369] [4. Bug Reports] [answered]
Hola

Quisiera saber si está correcto, debido a que tengo una compañía (PROD) que no sale lo que indica en la documentación.

adjunto imagen de la compañía
![image](https://user-images.githubusercontent.com/61419471/129277697-7787cb7d-92d0-47e3-914a-4de2985e4666.png)

adjunto link de la documentación
https://doc.cotalker.com/docs/documentation/admin/admin_company/

gracias =>

**Comment by guillevalin** [comment]
Gracias @NicolasPLG efectivamente es un bug.

**Comment by guillevalin** [comment, marked as answer]
Lo ingresaremos al pipeline de incidentes para que sea resuelto en el próximo release.

**Reply by NicolasPLG** [reply]
gracias @guillevalin
